<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Object</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>inputReturn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>Form</receiver>
   <slot>itemDblClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>Form</receiver>
   <slot>itemClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputReturn()</slot>
  <slot>itemDblClicked()</slot>
  <slot>itemClicked()</slot>
 </slots>
</ui>
